OpenActive.Server.NET/StoreBookingEngine/StoreBookingEngine.cs

```diff
@@ -117,11 +117,13 @@ public void SetResponseOrderItemAsSkeleton()
         public void SetResponseOrderItem(OrderItem item, SellerIdComponents sellerId, StoreBookingFlowContext flowContext)
         {
             if (item == null) throw new ArgumentNullException(nameof(item));
-            if (item?.OrderedItem.Object?.Id != RequestOrderItem?.OrderedItem.Object?.Id)
+            var requestOrderItemId = RequestOrderItem?.OrderedItem.Object != null ? RequestOrderItem.OrderedItem.Object.Id : RequestOrderItem?.OrderedItem.IdReference;
```

Comment:
OrderedItemId/AcceptedOfferId can still be an object because the proposal flow creates a full Order object.

Reply:
So it turns out the proposal flow creates this internally, so was able to remove